### PR TITLE
Add io.github.lainsce.Emulsion

### PR DESCRIPTION
**Pull request by lainsce**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by kevinsmia1939** [comment]
bot, build io.github.lainsce.Emulsion

**Comment by lainsce** [comment]
bot, build io.github.lainsce.Emulsion

**Comment by lainsce** [comment]
bot, build io.github.lainsce.Emulsion

File: io.github.lainsce.Emulsion.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Emulsion",
+    "rename-icon" : "io.github.lainsce.Emulsion",
+    "finish-args" : [
+        "--filesystem=xdg-download",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=session-bus",
```
Comment (barthalion) [inline io.github.lainsce.Emulsion.json:12]: Why does it need access to session bus?

Comment (lainsce) [reply io.github.lainsce.Emulsion.json:12]: When I removed this, it complained during my builds on Builder, and wouldn't even open the app.

**Comment by lainsce** [comment]
bot, build io.github.lainsce.Emulsion

File: io.github.lainsce.Emulsion.json (hunk 1 above)
Comment (bilelmoussaoui) [reply io.github.lainsce.Emulsion.json]: You're probably using the wrong app-id somewhere.

File: io.github.lainsce.Emulsion.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	{
+	    "name": "libsass",
```
Comment (bilelmoussaoui) [inline io.github.lainsce.Emulsion.json:29]: `cleanup: ["*"]` for both libsass & sassc

File: io.github.lainsce.Emulsion.json (hunk 1 above)
Comment (lainsce) [reply io.github.lainsce.Emulsion.json]: Doublechecked it, I think it was just me using the Devel in the app-id in my Builder builds. It runs without this, so fixed.

File: io.github.lainsce.Emulsion.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
+{
+    "app-id" : "io.github.lainsce.Emulsion",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Emulsion",
+    "rename-icon" : "io.github.lainsce.Emulsion",
+    "finish-args" : [
+        "--filesystem=xdg-download",
```
Comment (bilelmoussaoui) [inline io.github.lainsce.Emulsion.json:9]: what does it use xdg-download for?

Comment (lainsce) [reply io.github.lainsce.Emulsion.json:9]: That's for a future feature, exporting palettes.

Comment (bilelmoussaoui) [reply io.github.lainsce.Emulsion.json:9]: can be removed until needed, exporting palettes can use the native file chooser anyway

Comment (lainsce) [reply io.github.lainsce.Emulsion.json:9]: Alright

**Comment by lainsce** [comment]
bot, build io.github.lainsce.Emulsion

**Comment by lainsce** [comment]
Wait before merging this, I need to update it to 1.0.1, I found a bug.

**Comment by lainsce** [comment]
bot, build io.github.lainsce.Emulsion

**Comment by lainsce** [comment]
Anything else needed? :)

**Pull request closed without merging**